gdc_client/parcel/segment.py

```diff
@@ -95,7 +95,7 @@ def validate_segment_md5sums(self, path=None):
         log.debug("Checksumming {0}:".format(self.download.url))
 
         pbar = tqdm(
-            intervals, desc="Segment md5sum validation", file=sys.stdout, ascii=True,
+            iterable=intervals, desc="Segment md5sum validation", file=sys.stdout,
```

Comment:
This `tdqm` isn't a partial function. Do we still want to remove `ascii=True`?

Reply:
https://github.com/tqdm/tqdm#custom-integration

Reply:
Oh I see. I missed that import statement. Carry on. :eyes: